.changeset/rotten-birds-build.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-microsoft-calendar': patch
+---
+
+created new plugin @backstage/plugin-microsoft-calendar and added it to plugins directory
+updated example-app homepage to show microsoft-calendar widget.
```

Comment:
Can remove this part.

Reply:
I have changed the changeset, if it should be anything else, please let me know and elaborate on what do we need to write here in case of a new plugin.